### DISCUSSION TITLE
Event page content

### DISCUSSION THREAD

**Discussion by teamtacho** [liedekef/events-made-easy#256] [General]
How can I show any text on the event page, if somebody is going directly to the event page (e.g. https://..../events/)
The page is empty (as discribed) but that wasn't the case in older versions.

**Comment by liedekef** [comment]
The content of the special events page (as set in the EME options) is always changed by EME itself. Read this:
https://www.e-dynamics.be/wordpress/category/documentation/3-installation-and-usage/
